q2_metadata/_merge.py

```diff
@@ -19,21 +19,40 @@ def merge(metadata1: qiime2.Metadata,
     n_overlapping_ids = len(overlapping_ids)
     n_overlapping_columns = len(overlapping_columns)
 
-    if len(overlapping_ids) > 0 and len(overlapping_columns) > 0:
-        raise ValueError(f"Merging can currently handle overlapping ids "
-                         f"or overlapping columns, but not both. "
-                         f"{n_overlapping_ids} overlapping ids were "
-                         f"identified ({', '.join(overlapping_ids)}) and"
-                         f"{n_overlapping_columns} overlapping columns "
-                         f"were identified {', '.join(overlapping_columns)}.")
+    if n_overlapping_ids and n_overlapping_columns:
+        raise ValueError(
+            "Merging can currently handle overlapping ids or overlapping"
+            f"but not both. {n_overlapping_ids} overlapping ids were "
+            f"identified ({', '.join(overlapping_ids)}) and"
+            f"{n_overlapping_columns} overlapping columns were identified "
+            f"{', '.join(overlapping_columns)}."
```

Comment:
```suggestion
            f"({', '.join(overlapping_columns)})."
```
While we're at it, let's also make the listing of IDs and columns consistent - now both sets are listed in parentheses.